### ISSUE TITLE
v0.17 build missing dwarf

### ISSUE THREAD

**Issue by fbs** [iovisor/bpftrace#2492]
```
$ curl -O https://github.com/iovisor/bpftrace/releases/download/v0.17.0/bpftrace -L
$ sudo ./bpftrace --info
System
  OS: Linux 5.14.0-134.el9.x86_64 #1 SMP PREEMPT_DYNAMIC Thu Jul 21 12:57:06 UTC 2022
  Arch: x86_64

Build
  version: v0.17.0
  LLVM: 12.0.0
  unsafe uprobe: no
  bfd: yes
  libdw (DWARF support): no  <-------
```

seems like adding libdw is a nice addition to the build

**Comment by viktormalik** [comment]
Yeah, libdw is an optional dependency at the moment, but I guess that it's fine if we add it.

**Comment by viktormalik** [comment]
This seems to have been fixed for v0.18.0:
```
$ wget https://github.com/iovisor/bpftrace/releases/download/v0.18.0/bpftrace
$ sudo ./bpftrace --info
System
  OS: Linux 6.2.14-200.fc37.x86_64 #1 SMP PREEMPT_DYNAMIC Mon May  1 00:57:50 UTC 2023
  Arch: x86_64

Build
  version: v0.18.0
  LLVM: 12.0.0
  unsafe probe: no
  bfd: yes
  libdw (DWARF support): yes
```
Anything else needs to be done?
